support/sier-codec-js/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+use sier_codec::Parser;
+use wasm_bindgen::prelude::*;
+//TODO(melatron): Create a JS Class that holds the file_defs inside and
+// serialize/deserialize are methods of this class.
+#[wasm_bindgen]
+pub fn serialize(js_object: JsValue, file_defs: &str, struct_def: &str) -> js_sys::Uint8Array {
+    if !js_object.is_object() {
+        wasm_bindgen::throw_str("Provided argument is not a JS Object.");
```

Comment:
You might be able to have these functions return `Result<T, JsValue>` instead of explicitly throwing everywhere. More idiomatic and probably easier to read/write. Might even be able to use the `?` operator.